javascript/big-practice/src/scripts/views/noteView.js

```diff
@@ -21,7 +21,7 @@ export default class NoteView {
     noteElement.setAttribute('id', this.noteItem.id);
 
     noteElement.innerHTML = `
-      <label class="icon-check">
+      <label class="icon-check" data-id="${this.noteItem.id}">
```

Comment:
`this.noteItem` will be used many times when you binding data. Should catch a variable for it.